components/phone-field.tsx

```diff
@@ -11,12 +12,16 @@ export default function PhoneFields(props) {
       {phones
         ? phones.map((phone, index) => (
             <div className={styles.formRow} key={phone.key}>
-              <input
-                className={styles.formSoloInput}
-                type="tel"
-                name="number"
+              <PhoneInput
+                containerClass={styles.phoneContainer}
+                buttonClass={styles.phoneButton}
+                inputClass={styles.phoneInput}
+                dropdownClass={styles.phoneDropdown}
+                preferredCountries={["us", "ca"]}
+                placeholder={""}
+                country={phone.country_code || "us"}
```

Comment:
Isn't `country_code` an integer? How are we relating integers from Redtail into strings for this `PhoneInput` package? Have we tested loading a contact that has a country code of something outside the US?